src/reflect/scala/reflect/macros/Attachments.scala

```diff
@@ -76,6 +78,12 @@ abstract class Attachments { self =>
   def isEmpty: Boolean = true
 }
 
+object Attachments {
```

Comment:
```suggestion
private object Attachments {
```
Perhaps this may fix the Mima problem.